### ISSUE TITLE
 Settings - Background has green tint when side menu is opened

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#26405]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Go to staging.new.expensify,com
2. Open Settings page

## Expected Result:
The background does not have green tint

## Actual Result:
The background has green tint

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [ ] Windows / Chrome
- [ ] MacOS / Desktop

**Version Number:** v1.3.60-1

**Reproducible in staging?:** Yes

**Reproducible in production?:** No

**If this was caught during regression testing, add the test name, ID and link from TestRail:**

**Email or phone of affected tester (no customers):**

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/78819774/b07a0df3-9317-40a1-bec4-30df87926ad8

**Expensify/Expensify Issue URL:**

**Issue reported by:** Applause - Internal Team

**Slack conversation:**

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by ishpaul777** [comment]
## Proposal 

## ROOT CAUSE
We were setting the overlay colours using getBackgroundColorWithOpacityStyle before navigation refactor in #22437
```
StyleUtils.getBackgroundColorWithOpacityStyle(themeColors.shadow, CONST.RIGHT_MODAL_BACKGROUND_OVERLAY_OPACITY),
```
After refactor ther overlay directly uses
```
 backgroundColor: themeColors.overlay,
```

## Changes
revert back to using the getBackgroundColorWithOpacityStyle for backgroundcolour

**Comment by mountiny** [comment]
I dont think this is a blocker and can be handled in this existing issue https://github.com/Expensify/App/issues/26390
